packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapterInit.ts

```diff
@@ -33,6 +35,10 @@ export const initialize = ({
   testPath,
 }: {
   config: Config.ProjectConfig;
+  environment: JestEnvironment & {
+    // Move this into the JestEnvironment type as an optional method when jest-circus is default.
```

Comment:
No reason to wait, is there?

Reply:
It'll add a `jest-circus` dependency to basically everyone when it has very limited use today. I didn't think it was worth the trade off until it's default.

You think it is?

Reply:
Not if it's optional. Or do you mean the test events types? Might make sense to stick them in `@jest/types`?

Reply:
That's possible. I hesitated because the `Event` and `State` types are extremely core to Circus so we'd be moving a large chunk over. I can see the value, though, and I think you're right. I'll do that.

Reply:
Done :)